Tiltfile

```diff
@@ -115,6 +115,26 @@ COPY --from=tilt-helper /usr/bin/kubectl /usr/bin/kubectl
     },
 }
 
+# Create a data structure to hold information about addons.
+addons = {
+    "test-extension": {
+        "context": "./test/extension",
+        "image": "gcr.io/k8s-staging-cluster-api/test-extension",
+        "container_name": "extension",
+        "binary_name": "extender",
+        "namespace": "default",
+        "live_reload_deps": ["main.go", "handlers"],
+        "label": "RuntimeSDK",
+        "kustomize_substitutions": {
+            "SERVICE_NAMESPACE": "default",
+        },
+        "additional_resources": {
+            "ExtensionConfig": "config/tilt/extensionconfig.yaml",
```

Comment:
Q: do we actually have to look for a specific kind and only deploy that or is there a way to just deploy the entire YAML?
Just deploying the entire YAML feels a lot simpler and more intuitive